src/style.css

```diff
@@ -1,3 +1,79 @@
 body {
   margin: 0;
 }
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  padding: 0 50px;
+
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo {
+  display: block;
+  height: 40px;
+  width: 40px;
+}
+
+.logo__image {
+  height: 100%;
+  width: 100%;
+}
+
+.nav__list {
+  display: flex;
+  padding: 0;
+  margin: 0;
+  list-style: none;
+}
+
+.nav__item {
+  margin-left: 20px;
```

Comment:
Be consistent with your vertical margins. It's recommended to add only top or only bottom margin, but not both. This helps to avoid issues with collapsing margins. In this case, consider using either `margin-top` or `margin-bottom` for the `.nav__item` class, but not `margin-left` which is horizontal.